src/main/java/org/opentripplanner/routing/api/request/WheelchairAccessibilityRequest.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.routing.api.request;
+
+public record WheelchairAccessibilityRequest(
+  boolean enabled,
+  WheelchairAccessibilityFeature trips,
+  WheelchairAccessibilityFeature stops
+) {
+  public static final WheelchairAccessibilityRequest DEFAULTS = new WheelchairAccessibilityRequest(
```

Comment:
Rename to `DEFAULT`, is is a single object not a collection.

Reply:
Done.